src/routes/(index)/components/Dashboard.svelte

```diff
@@ -60,6 +59,18 @@
 			return override;
 		});
 	};
+	const onUpdateShowHiddenFiles = (newValue: string) => {
+		if (userData.showHiddenFiles === newValue) return;
+		if (firebaseUser && firebaseUser.uid) {
+			userData.showHiddenFiles = newValue as 'yes' | 'no';
+			console.log(userData);
+			update(ref(database, `users/${firebaseUser.uid}/data`), userData)
```

Comment:
Rather than updating `/data`, can we update `/data/showHiddenFiles` instead, setting the value to either "yes" or "no"?